lib/ssl/doc/src/ssl.xml

```diff
@@ -213,6 +225,14 @@
       Defaults to <c>tls</c> Introduced in OTP 20, DTLS support is considered
       experimental in this release. DTLS over other transports than UDP are not yet supported.</p></item>
```

Comment:
... **is** not yet supported.

Reply:
Transports is plural so I think it should  be are?!

Reply:
It depends on whether you consider that it is "DTLS over other transport" (singular) or "other transports" (plural). By the structure of the sentence, I'd agree with singular as it's a general structure that is not there, but if it's the other way around, I'd recommend rewording like:

> Other transports than UDP are not yet supported for DTLS.

Reply:
I will go for the rewording. The other transport most desired at the moment I think is SCTP but I think it is a good practice to not make limiting assumptions.